draft-ietf-quic-transport.md

```diff
@@ -1068,9 +1069,12 @@ has a non-zero-length Destination Connection ID corresponding to an existing
 connection, QUIC processes that packet accordingly. Note that more than one
 connection ID can be associated with a connection; see {{connection-id}}.
 
-If the Destination Connection ID is zero length and the packet matches the
-local address and port of a connection where the endpoint used zero-length
-connection IDs, QUIC processes the packet as part of that connection.
+If the Destination Connection ID is zero length and the addressing information
+in the packet matches the addressing information the endpoint uses to identify a
+connection with a zero-length connection ID, QUIC processes the packet as part
+of that connection.  An endpoint can use just destination IP and port or both
```

Comment:
I like the rewording of the first sentence, but I don't think I like the new second sentence. This somewhat encourages folks to only use destination IP and port with zero-length connection IDs, and I think this might cause issues for readers that don't go look up the serious limitations that are somewhat buried in the Connection ID section. What's the purpose of the second sentence?

Reply:
Conversely, I find "addressing information" to be vague, and the second sentence is much clearer.

Reply:
Would making the text more alarming help?  s/subject to the limitations described in XX/though this makes connections fragile as described in XX/

Reply:
Making the text more alarming would resolve my concern. Thanks!